client/src/legacy/LeftAndMain.EditForm.js

```diff
@@ -569,7 +569,23 @@ $.entwine('ss', function($){
       }
     },
     onchange: function (e) {
-      this.toggleListDisplay(e.target.value);
+      this.toggleListDisplay(e.target.value, window.matchMedia('(prefers-reduced-motion: reduce)').matches);
```

Comment:
Technically out of scope but this just makes the animation not happen if the user as set their settings to prefer reduced motion. I can split this out if it's adding significant review overhead but I imagine it shouldn't do.